services/libssh/cve_2018_10933_server.py

```diff
@@ -0,0 +1,338 @@
+# -*- coding: utf-8 -*-
+#!/usr/bin/env python # noqa: E265
+"""libssh Honeycomb Service with CVE-2018-10933 support."""
+from __future__ import unicode_literals
+
+import socket
+import sys
+import os
+import threading
+import struct
+from six.moves.socketserver import ThreadingTCPServer, StreamRequestHandler
+
+import paramiko
+from paramiko import util
+from paramiko import Message
+from paramiko.py3compat import b, long
+from paramiko.transport import _active_threads
+from paramiko.common import (
+    xffffffff,
+    MSG_KEXINIT,
+    DEBUG,
+    ERROR,
+    WARNING,
+    MSG_IGNORE,
+    MSG_DISCONNECT,
+    MSG_DEBUG,
+    MSG_NAMES,
+    MSG_UNIMPLEMENTED,
+    MSG_USERAUTH_SUCCESS,
+    cMSG_UNIMPLEMENTED,
+)
+from paramiko.ssh_exception import SSHException
+from paramiko.packet import NeedRekeyException
+
+from consts import SERVER_SIG, EVENT_TYPE_FIELD_NAME, SSH_ALERT_TYPE, USERNAME_FIELD_NAME, PASSWORD_FIELD_NAME,\
+    ADDITIONAL_FIELDS_FIELD_NAME, KEY_FIELD_NAME, CVE_SSH_PORT
+
+# setup logging
+paramiko.util.log_to_file("demo_server.log")
+
+# Get current rsa key, or generate new one if needed
+key_file_path = os.path.join(os.path.dirname(__file__), "libssh_server_rsa.key")
+if not os.path.exists(key_file_path):
+    host_key = paramiko.RSAKey.generate(1024)
+    try:
+        host_key.write_private_key_file(key_file_path)
+    except:  # noqa: E722
```

Comment:
this is new code, no reason not to catch an Exception like a normal human being..
```suggestion
    except Exception:
```